Makefile

```diff
@@ -1,6 +1,6 @@
 # variables that should not be overridden by the user
-VERSION = edge
-SELF_DIR := $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
+VERSION = 1.5.0
+SELF_DIR := $(dir $(lastword $(MAKEFILE_LIST)))
```

Comment:
Why this change?

Reply:
This was after I rebased after your merge. Maybe a bad merge. Let me recheck